src/styles/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
```

Comment:
In the task was told: "Reuse stars block from [Stars task]" (https://github.com/mate-academy/layout_stars)'.
Border-radius was added to that block.  https://www.figma.com/file/EIBkG1dy1jnK88YPO34Qir/Moyo-%2F-Catalog-(updated)?node-id=11325%3A2961&t=MyQNUIAgRCGo4BEC-0

Reply:
Perhaps, first time mentor didn't notice that you had used border-radius. 
I checked a figma maket. And it doesn't make sense to add border-radius here as [OlEzhA0](https://github.com/OlEzhA0) said 
Because you use star svg that already has rounded angles. 
Just be attentive. Not all styles that you get at figma you need to copy into your css

Reply:
understood.
 thanks